### PR TITLE
Lesson 3 - 6 tasks

### PR DESCRIPTION
**Pull request by karinanartowska**

### CODE REVIEW

**Comment by karinanartowska** [comment]
@gradopaw please review :)

File: students/nartowska_karina/lesson_03_functions/calculator.py [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+            help()
+        elif option == "q":
+            quit_the_program()
+            break
+        else:
+            print("Enter option again: ")
+            continue
+
+
+main()
```
Comment (gradopaw) [inline students/nartowska_karina/lesson_03_functions/calculator.py]: Please add:
if __name__ == "__main__":
    main()

Using that Python files can act as either reusable modules or as standalone programs

Have a look at:
http://thepythonguru.com/what-is-if-__name__-__main__/

File: students/nartowska_karina/lesson_03_functions/exponentiation.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    print("Enter a positive real number 'a' and a non-negative integer 'n'")
+    print("a is: ")
+    a = float(input())
+    print("n is: ")
+    n = int(input())
+    print("Result:")
+    print(power(a, n))
+
+
+main()
```
Comment (gradopaw) [inline students/nartowska_karina/lesson_03_functions/exponentiation.py]: See comment to calculator.py

File: students/nartowska_karina/lesson_03_functions/fibonacci_numbers.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+def main():
+    print("Enter a non-negative integer 'n'")
+    print("n is: ")
+    n = int(input())
+    print("Result:")
+    print(fib(n))
+
+
+main()
```
Comment (gradopaw) [inline students/nartowska_karina/lesson_03_functions/fibonacci_numbers.py]: See comment to calculator.py

File: students/nartowska_karina/lesson_03_functions/negative_exponent.py [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+    print("Enter a positive real number 'a' and integer 'n'")
+    print("a is: ")
+    a = float(input())
+    print("n is: ")
+    n = int(input())
+    print("Result:")
+    power(a, n)
+
+
+main()
```
Comment (gradopaw) [inline students/nartowska_karina/lesson_03_functions/negative_exponent.py]: See comment to calculator.py

File: students/nartowska_karina/lesson_03_functions/the_collatz_sequence_and_input_validation.py [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+
+
+def main():
+    try:
+        input_collatz()
+    except ValueError:
+        print("Value must be an integer")
+
+
+main()
```
Comment (gradopaw) [inline students/nartowska_karina/lesson_03_functions/the_collatz_sequence_and_input_validation.py]: See comment to calculator.py

File: students/nartowska_karina/lesson_03_functions/the_length_of_the_segment.py [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+    y1 = float(input())
+    print("x2 is: ")
+    x2 = float(input())
+    print("y2 is: ")
+    y2 = float(input())
+    print("The distance is:")
+    distance(x1, y1, x2, y2)
+
+
+main()
```
Comment (gradopaw) [inline students/nartowska_karina/lesson_03_functions/the_length_of_the_segment.py]: See comment to calculator.py

**Comment by karinanartowska** [comment]
@gradopaw please review commit "Changes after review" :)

**Pull request merged**